package.json

```diff
@@ -116,7 +117,7 @@
     "rimraf": "^6.0.1",
     "rollup": "^4.20.0",
     "typescript": "~5.5.4",
-    "vitest": "1.2.2",
+    "vitest": "^2.0.5",
```

Comment:
vitest v2 ([changelog](https://github.com/vitest-dev/vitest/releases/tag/v2.0.0)) enables `coverage.ignoreEmptyLines` by default so coverage is expected to drop slightly in some files